flowpm/fourier_smoothing.py

```diff
@@ -0,0 +1,61 @@
+from scipy.stats import norm
+import numpy as np
+import tensorflow_addons as tfa
+import tensorflow as tf
+import flowpm
+
+
+def make_power_map(power_spectrum, size, kps=None):
+  #Ok we need to make a map of the power spectrum in Fourier space
+  k1 = np.fft.fftfreq(size)
+
+  k2 = np.fft.fftfreq(size)
+
+  kcoords = np.meshgrid(k1, k2)
+
+  # Now we can compute the k vector
+  k = np.sqrt(kcoords[0]**2 + kcoords[1]**2)
+
+  if kps is None:
+    kps = np.linspace(0, 0.5, len(power_spectrum))
+  # And we can interpolate the PS at these positions
+
+  ps_map = np.interp(k.flatten(), kps, power_spectrum).reshape([size, size])
+
+  ps_map = tf.cast(ps_map, dtype=tf.float32)
+  return ps_map
+
+
+def fourier_smoothing(im, sigma, resolution, source_plane=None):
```

Comment:
it's not explained what this `source_plane` keyword does. The name at least seems to be very specific, but I think what it does is just whether or not to return the Fourier transform of the image